src/shared/frame-rpc.js

```diff
@@ -53,46 +60,67 @@ const VERSION = '1.0.0';
 export class RPC {
   /**
    * Create an RPC client for sending RPC requests from `sourceFrame` to
-   * `destFrame`.
+   * `destFrame`, and receiving RPC responses from `destFrame` to `sourceFrame`.
    *
-   * @param {Window} sourceFrame
-   * @param {Window} destFrame
-   * @param {string} origin - Origin of destination frame
-   * @param {Record<string, (...args: any[]) => any>} methods - Map of method
+   * This class has been adapted to work with `MessageChannel`. Because
+   * `MessageChannel` are unidirectional we ignore the `sourceFrame` and `origin`
```

Comment:
`MessageChannel` is not unidirectional. That would suggest messages can only be sent on one port and received on the other port. I think what you mean here is that `MessageChannel`s only support receiving events from a single sender which can't move between origins unless explicitly transferred by the current owner (unlike the `Document`/`window` relationship).